cips/cip-14.md

```diff
@@ -213,7 +213,18 @@ This proposal is backwards-incompatible because it is state-machine breaking. Th
 
 ## Test Cases
 
-TBA
+The following test cases are available in the [ibc-go e2e repository.](https://github.com/cosmos/ibc-go/tree/main/e2e/tests/interchain_accounts)
+
+- Registration of an interchain account
```

Comment:
Ref: https://github.com/cosmos/ibc-go/blob/1ebede6c7d4d88f99b9046fa9e40db6571b46b81/e2e/tests/interchain_accounts/upgrades_test.go#L249-L259

Reply:
would you prefer that I put a link for each test case scenario rather than the current folder?

Reply:
Yea links sound helpful! Especially if we'd like these exact test cases to be imported and/or copy + pasted from ibc-go into the celestia-app repo when we implement ICA.